### PR TITLE
Add piloter

### PR DESCRIPTION
**Pull request by clareconstantine**
# Description

Allow admins to add teachers to the CSD and CSP pilots from the admin pages, instead of requiring engineering help.

Screen Shot 2019-11-07 at 1 12 21 PM

Success:
Screen Shot 2019-11-07 at 10 35 12 AM

Error when trying to add a student:
Screen Shot 2019-11-07 at 10 40 39 AM

Error when adding an email not associated with an account:
Screen Shot 2019-11-07 at 10 41 39 AM

## Links

- [LP-804](https://codedotorg.atlassian.net/browse/LP-804?atlOrigin=eyJpIjoiOWRmMzFjNzBjNTdjNDIwOThhMTBiMmJmN2Y4ZWEzMjQiLCJwIjoiaiJ9)

## Testing story

Added unit tests

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/controllers/admin_search_controller.rb [hunk 1]
```diff
@@ -79,4 +79,19 @@ def show_pilot
     user_ids =  SingleUserExperiment.where(name: @pilot_name).map(&:min_user_id)
     @emails = User.where(id: user_ids).pluck(:email)
   end
+
+  def add_to_pilot
+    email = params[:email]
+    pilot_name = params[:pilot_name]
+    user = User.find_by_email_or_hashed_email(email)
```
Comment (davidsbailey) [inline dashboard/app/controllers/admin_search_controller.rb:87]: please add this here, copied from above: `return head :bad_request unless Experiment::PILOT_EXPERIMENTS.include?(pilot_name)`

**Review by davidsbailey — APPROVED** [review]
Looks great, @clareconstantine ! Just one request below.

**Review by dmcavoy — APPROVED** [review]
🎉

**Review by Erin007 — APPROVED** [review]
Yay for fishing pole features!

**Pull request merged**